src/pages/userPage/Blanks/Blanks.tsx

```diff
@@ -191,16 +196,16 @@ export const Blanks = () => {
                                         {(document.fileName?.length > fileNameMaxLength) ?
                                             <Tooltip className={classes.antTooltipInner} title={document.fileName}>
                                                 <Paragraph ellipsis={{ rows: 2, suffix: " " }}>
-                                                    {document.fileName.slice(0,fileNameMaxLength-1) + "..."}
+                                                    {document.fileName.slice(0, fileNameMaxLength - 1) + "..."}
                                                 </Paragraph>
                                             </Tooltip>
-                                        : <Paragraph ellipsis={{ rows: 2, suffix: " " }}>
-                                            {document.fileName}
-                                          </Paragraph>
+                                            : <Paragraph ellipsis={{ rows: 2, suffix: " " }}>
+                                                {document.fileName}
+                                            </Paragraph>
                                         }
 
                                     </div>
-                                    {(userToken.nameid === userId || IsUserHasAccessToManageBlanks(roles)) &&
+                                    {(userToken.nameid === userId || IsUserHasAccessToSeeAndDownloadBlanks(roles)) &&
```

Comment:
'IsUserHasAccess...'? Maybe it'd be better 'DoesUserHaveAccess...'

Reply:
Yes, much more. Done